aries-site/src/pages/components/checkbox.js

```diff
@@ -24,11 +32,139 @@ const CheckBox = () => (
         </SubsectionText>
         <Example
           docs="https://v2.grommet.io/checkbox?theme=hpe#props"
-          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/CheckboxExample.js"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/checkbox/CheckboxSimpleExample.js"
+          figma="https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1745%3A43"
+          designer="https://designer.grommet.io/home?id=HPE-design-system-taylor-seamans-hpe-com"
+        >
+          <CheckBoxSimpleExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBox can be used to present an individual option or group of
+          options to the user. When presented in a group, the user can select a
+          single or multiple options. Refer to CheckBoxGroup for examples.
+        </SubsectionText>
+        <SubsectionText>
```

Comment:
Maybe these next two paragraphs could fall under the "About CheckBox" section, but we could rename it to "Usage" like this page? https://design-system.hpe.design/components/tabs#usage